app/services/billable_metrics/update_service.rb

```diff
@@ -44,12 +53,10 @@ def call
     attr_reader :billable_metric, :params
 
     def update_groups(metric, group_params)
-      ActiveRecord::Base.transaction do
-        Groups::CreateOrUpdateBatchService.call(
-          billable_metric: metric,
-          group_params: group_params.with_indifferent_access,
-        )
-      end
+      Groups::CreateOrUpdateBatchService.call(
+        billable_metric: metric,
+        group_params: group_params.with_indifferent_access,
+      )
```

Comment:
```suggestion
      ).raise_if_error!
```

Reply:
No, the error result is handled differently here. I agree we should use `raise_if_error!` but since the group related logic will be deprecated, I don't want to invest time in refactoring the way it's working now

Reply:
Allright!